docs/docs/gatsby-style-guide.md

```diff
@@ -278,3 +289,154 @@ The Hemingway App will assign a “grade level” for your writing. You should a
+- error messages that tell them if it’s a known bug/issue, lead them to docs, and/or suggest fixes
+- guides for how things work in Gatsby
+- they usually already have strong opinions or requirements about what they want to use as their CMS or data source and want to know best practices for their workflow
+- signs that Gatsby is a reliable, long-term choice (signs that it is growing and improving and evidence it will be around for a long time)
+- ways to check their project’s requirements against what Gatsby offers
+- open source code from well-built example sites
+
+### Docs purpose
+
+By referencing the docs, a user should:
+
+- get tasks done as quickly as possible
+- evaluate options for getting tasks done as quickly as possible
+- build sites and apps as quickly as possible, including the following kinds of sites:
+  - marketing
+  - blogs
+  - portfolio
+  - e-commerce
+  - authentication
+
+### Docs tone and style
+
+The tone and style of the docs should effectively help the audience reach their goals.
+
+### Use personal "you"
+
+The docs use the personal “you” to address the user.
+
+#### Give experts as much relevant info to get the task done as quickly as possible
+
+Because the audience of the docs is people who have intermediate to expert level of mastery with React, it's important to provide the information needed to complete tasks in addition to all relevant and helpful context, references,and alternatives. The goal: give people the information necessary to get tasks done as quickly and effectively as possible.
+
+In practice, you can reach this goal by two rules of thumb:
+
+- Include an "additional information" section at the bottom of each guide with hyperlinks to relevant external blogposts, tutorials, and other Gatsby resources and docs.
+- When there are multiple ways to complete a task, explain the following:
+
+1.  The most common way of accomplishing the task
+2.  The best way to accomplish the task on the lowest supported versions of software
+3.  The best practice and why is it the best (if different than 3)
+4.  Any tips on how to pick an option
+
+## KPIs
+
+Here is how we measure the quality of the tutorial and docs. We will use ([cohort analysis](https://www.geckoboard.com/learn/kpi-examples/mobile-app-kpis/retention-rate/) to track improvements or regressions to the tutorial and docs.
+
+### Tutorial
+
+- time to value: how quickly, effectively, and painlessly can users experience the value of Gatsby?
+- daily, weekly, monthly active users
+- daily, weekly, monthly “finishers” of the tutorial (finisher = people who spend 5 mins minimum on each page of tutorial and go through whole thing)
+- weekly retention rate
+
+### Docs
+
+- time to task: how quickly and effectively can users accomplish tasks?
+- daily, weekly, monthly active users
+- weekly retention rate
+
+## Goals of docs and tutorial maintenance
+
+The way the Gatsby community maintains docs and tutorials must meet the following requirements:
```

Comment:
Is the following list saying how we evaluate and maintain our docs? What happens if something doesn't meet the requirements? I don't think I understand what this is saying.

Reply:
Hm, yeah, this is kind of vague. I meant to write these things down to outline standards for how we write and maintain docs (process-oriented stuff) so that if anyone months or years down the road takes over docs and I'm not involved, they know why we made the choices we did. I just updated it in response to your question. Is it clearer? Any suggestions?

Reply:
Yes, that makes more sense!